lib/utils/deprecations.js

```diff
@@ -8,3 +8,4 @@ exports.noRawAttributes = deprecate(noop, 'Use sequelize.fn / sequelize.literal
 exports.noTrueLogging = deprecate(noop, 'The logging-option should be either a function or false. Default: console.log', 'SEQUELIZE0002');
 exports.noStringOperators = deprecate(noop, 'String based operators are deprecated. Please use Symbol based operators for better security, read more at http://docs.sequelizejs.com/manual/querying.html#operators', 'SEQUELIZE0003');
 exports.noBoolOperatorAliases = deprecate(noop, 'A boolean value was passed to options.operatorsAliases. This is a no-op with v5 and should be removed.', 'SEQUELIZE0004');
+exports.noDoubleNestedGroup = deprecate(noop, 'Passing a double nested nested array to `group` is unsupported and will be removed in v6.', 'SEQUELIZE0005');
```

Comment:
This pattern is supported everywhere, do you plan to remove this from all other segments, for now this looks like it is only deprecated from `aggregate` methods

Reply:
Does it also work in `findAll` etc?